lib/go-twitter/README.md

```diff
@@ -53,7 +53,7 @@ tweets, resp, err := client.Timelines.HomeTimeline(&twitter.HomeTimelineParams{
 })
 
 // Send a Tweet
-tweet, resp, err := client.Statuses.Update("just setting up my twttr", nil)
```

Comment:
This is intentional; the original tweet referenced here has this spelling. See <https://twitter.com/jack/status/20>.